draft-ietf-quic-transport.md

```diff
@@ -1771,20 +1772,18 @@ the expectation that either the network or the peer no longer supports ECN.
 To protect the connection from arbitrary corruption of ECN codepoints by the
 network, an endpoint verifies the following when an ACK_ECN frame is received:
 
-* The total increase in ECT(0), ECT(1), and CE counters reported in the ACK_ECN
-  frame MUST be equal to the total number of packets newly acknowledged in this
-  ACK_ECN frame.
-
-* The increase in ECT(0) and ECT(1) counters MUST be no greater than the number
+* The increase in ECT(0) and ECT(1) counters MUST be no lesser than the number
```

Comment:
"no lesser" isn't a turn of phrase I'm comfortable with.  "MUST be at least the number of newly acknowledged packets"